cilium-dbg/cmd/cleanup.go

```diff
@@ -573,8 +573,13 @@ func removeTCFilters(linkAndFilters map[string][]*netlink.BpfFilter) error {
 }
 
 func removeXDPAttachments(links []netlink.Link) error {
+	params := loader.Params{
```

Comment:
Very ugly, but we've got follow-ups coming right after that clean this up.

Reply:
This will become impossible to instantiate in `cilium-dbg` pretty soon, but it's fine for now.